wazimap/templates/_base.html

```diff
@@ -79,10 +79,10 @@
                     {% endblock %}
                 {% endif %}
                 <div class="header-buttons">
+                {% block header_branding %}
```

Comment:
Hmm, this will probably break anyone who's overriding this header. :(

Can we instead make the bit around the logo a block, and override that in wazimap-za?